sample/src/main/scala/za/co/absa/spline/sample/batch/CodelessInitSampleJob.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.sample.batch
+
+import org.apache.spark.sql.SaveMode
+import za.co.absa.spline.sample.SparkApp
+
+/**
+  * A sample job where no explicit Spline code was used to initialize Spline. Only Spark configuration was used.
+  */
+object CodelessInitSampleJob extends SparkApp(
+  name ="Codeless Init Sample Job",
+  // Spark configuration used to register Spline listener for codeless init.
+  conf = Seq(("spark.sql.queryExecutionListeners", "za.co.absa.spline.core.listener.CodelessQueryExecutionListener"))) {
```

Comment:
I would avoid using the word 'codeless' in the execution listener class name. It sounds confusing. Can we rename it to just `QueryExecutionListener`?

Reply:
I don't think ```QueryExecutionListener``` is a good idea as it would be duplicating name of ```QueryExecutionListener``` although in different package. At least we should suffix it with ```Impl```.
It would be better to give the class name that identifies its purpose. Here I am assuming that we will call the feature "codeless init", thus the name. I would be possibly even better to call it ```CodelessInitQueryExecutionListener```, so something akin that.